app/scripts/lib/config-loader.js

```diff
@@ -58,16 +56,16 @@ define([
         // did not receive the cookie.
       }
 
-      self._configPromise = self._xhr.getJSON('/config')
-        .fail(function (jqXHR) {
+      this._configPromise = this._xhr.getJSON('/config')
+        .fail((jqXHR) => {
           throw ConfigLoader.Errors.normalizeXHRError(jqXHR);
         });
 
-      return self._configPromise;
-    },
-  };
+      return this._configPromise;
+    }
+  }
 
-  var t = function (msg) {
+  let t = function (msg) {
```

Comment:
Good point– jsxgettext would need to support ES6 also or we can try extraction on babel's output. If the latter case, we'd need to guard against variable renaming of the `t` function by e.g. using a method instead (`Strings.t('...')`).